tests/loginValidUser.test.js

```diff
@@ -0,0 +1,21 @@
+const { test, expect } = require('@playwright/test');
+const { LoginPage } = require('../pages/loginPage.js');
+const { HomePage } = require('../pages/homePage.js');
+
+
+
+test.describe('Redmine login', () => {
+    test('should login with existing user', async ({ page }) => {
+        const homePage = new HomePage(page);
+        const loginPage = new LoginPage(page);
+
+        await loginPage.goto('https://www.redmine.org/login');
+        const user = {
+            username: 'Brgdpfv',
+            password: 'adgjadgj',
+        };
+        await loginPage.loginUser(user);
```

Comment:
@karenn32 Here is goes the same as for the 'should not login with random user and password' (that is why menthioning the test case ID is important).